From aa4043875497431cf6b13b275e89e271eec706b4 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Fri, 30 Sep 2016 09:28:14 -0400
Subject: [PATCH] Add path for libstemmer headers to be found

---
 src/CMakeLists.txt | 17 +++++++++++++++--
 1 file changed, 15 insertions(+), 2 deletions(-)

diff -Naur AppStream-0.10.2/src/CMakeLists.txt AppStream-0.10.2.tpg/src/CMakeLists.txt
--- AppStream-0.10.2/src/CMakeLists.txt	2016-10-10 17:53:28.000000000 +0000
+++ AppStream-0.10.2.tpg/src/CMakeLists.txt	2016-10-18 18:49:00.089068008 +0000
@@ -11,6 +11,8 @@
 	find_library(SNOWBALL_LIB stemmer)
 	if (NOT SNOWBALL_LIB)
 		message (FATAL_ERROR "Stemming support is enabled, but libstemmer was not found. Please install the Snowball development files.")
+	else ()
+		set(LIBSTEMMER_INCLUDE_DIR "/usr/include/libstemmer")
 	endif ()
 else ()
 	set(SNOWBALL_LIB "")
@@ -99,13 +101,23 @@
 		as-stemmer.h
 )
 
-include_directories(${CMAKE_BINARY_DIR}
+if (STEMMING)
+    include_directories(${CMAKE_BINARY_DIR}
 		    ${CMAKE_BINARY_DIR}/src
 		    ${GLIB2_INCLUDE_DIRS}
 		    ${GIO2_INCLUDE_DIRS}
 		    ${LIBXML2_INCLUDE_DIR}
 		    ${YAML_INCLUDE_DIR}
-)
+		    ${LIBSTEMMER_INCLUDE_DIR}
+		    )
+else ()
+    include_directories(${CMAKE_BINARY_DIR}
+		    ${CMAKE_BINARY_DIR}/src
+		    ${GLIB2_INCLUDE_DIRS}
+		    ${GIO2_INCLUDE_DIRS}
+		    ${LIBXML2_INCLUDE_DIR}
+		    ${YAML_INCLUDE_DIR}
+endif ()
 
 add_definitions("-DAS_COMPILATION")
 
